src/podman.scss

```diff
@@ -21,10 +21,8 @@
 }
 
 .container-block small {
-    flex-basis: calc(var(--pf-global--LineHeight--sm) * 1rem);
-    line-height: var(--pf-global--LineHeight--sm);
-    opacity: 0.7;
-    overflow: hidden;
+    @include pf-line-clamp("1");
```

Comment:
Thanks - much nicer.

Reply:
Thanks for pointing it out! :+1: 

It basically does the same thing after compilation, but it is so much nicer in the source code here. Plus, I removed the rest of our previous bad hack (which I didn't completely remove before).